src/index.ts

```diff
@@ -19,37 +19,62 @@ const run: Edkt['run'] = async (config) => {
     if (!hasConsent) return;
   }
 
-  // This is a no-op if undefined, equals current value or lesser than 0
+  // This is a no-op if undefined or lesser than 0
   viewStore.setStorageSize(featureStorageSize);
   viewStore.insert(pageFeatures, pageMetadata);
 
-  const matchedAudiences = audienceDefinitions
+  const currentTS = timeStampInSecs();
+
+  const versionMatchPredicate = (
```

Comment:
I suggest to rename to prefix with is, does etc and remove the Predicate suffix, eg: `doesAudienceVersionMatch()`